### PR TITLE
Label util

### PR DESCRIPTION
**Pull request by AdamSaleh**

### CODE REVIEW

File: vars/getPullRequestLabels.groovy [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        withCredentials([usernamePassword(
+                    credentialsId: gitHubCredentialsId,
+                    passwordVariable: 'GITHUB_PASSWORD',
+                    usernameVariable: 'GITHUB_USERNAME')]) {
+            final GitHub gitHub = new GitHubBuilder()
+                .withOAuthToken(env.GITHUB_PASSWORD, env.GITHUB_USERNAME)
+                .build()
+
+            final GHOrganization ghOrganization = gitHub.getOrganization(org)
+            def labels = ghOrganization.getRepository("mobile-cli").getPullRequest(72).getLabels();
```
Comment (vchepeli) [inline vars/getPullRequestLabels.groovy]: why 72 here?

Comment (ghost) [inline vars/getPullRequestLabels.groovy]: why 72 here? should be pr

**Review by ghost — CHANGES_REQUESTED** [review]

File: vars/getPullRequestLabels.groovy (hunk 1 above)
Comment (AdamSaleh) [reply vars/getPullRequestLabels.groovy]: Thanks! That was my placeholder, and I didn't notice I forgot to replace it :)

File: vars/getPullRequestLabels.groovy [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        withCredentials([usernamePassword(
+                    credentialsId: gitHubCredentialsId,
+                    passwordVariable: 'GITHUB_PASSWORD',
+                    usernameVariable: 'GITHUB_USERNAME')]) {
+            final GitHub gitHub = new GitHubBuilder()
+                .withOAuthToken(env.GITHUB_PASSWORD, env.GITHUB_USERNAME)
+                .build()
+
+            final GHOrganization ghOrganization = gitHub.getOrganization(org)
+            def labels = ghOrganization.getRepository("mobile-cli").getPullRequest(pr).getLabels();
```
Comment (ghost) [inline vars/getPullRequestLabels.groovy]: "mobile-cli" -> repository

**Review by ghost — CHANGES_REQUESTED** [review]

File: vars/getPullRequestLabels.groovy [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    def config = [:]
+    body.resolveStrategy = Closure.DELEGATE_FIRST
+    body.delegate = config
+    body()
+
+    def prnfo = getPullRequestInfo()
+    def org = config.organization ?: prnfo["org"]
+    def repo = config.repository ?: prnfo["repo"]
+    def pr = config.pr ?: prnfo["id"]
+    def credentials = config.credentials ?: "githubjenkins"
```
Comment (ghost) [inline vars/getPullRequestLabels.groovy:50]: "githubjenkins" -> gitHubCredentialsId

**Review by ghost — CHANGES_REQUESTED** [review]

File: vars/getPullRequestLabels.groovy (hunk 3 above)
Comment (AdamSaleh) [reply vars/getPullRequestLabels.groovy:50]: This one is actually correct :-) That is the default to be used, similarly as in enforceTrustedApproval

File: vars/getPullRequestLabels.groovy (hunk 2 above)
Comment (AdamSaleh) [reply vars/getPullRequestLabels.groovy]: Thanks!

**Review by ghost — APPROVED** [review]

**Pull request merged**